PiwikTracker/Tracker.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+
+final public class Tracker: NSObject {
+    
+    let dispatcher: Dispatcher
+    var queue: Queue
+    let siteId: String
+    
+    required public init(siteId: String, queue: Queue, dispatcher: Dispatcher) {
+        self.siteId = siteId
+        self.queue = queue
+        self.dispatcher = dispatcher
+        super.init()
+        startDispatchTimer()
+    }
+    
+    private func queue(event: Event) {
+        queue.queue(item: event) { 
+            // Discussion: Do we want the possibility to dispatch synchronous?
```

Comment:
The old SDK hat an option to enable synchronous or rather direct dispatching. So once an event was added to the queue it was immediately dispatched. I do not really to add that feature again. It adds some complexity to the whole code and mostly the same effect can be accomplished by setting the dispatch interval as low as preferred.

If we remove the possibility to transmit Events directly, I would like to remove the Dispatcher.event that transmits only a single event and always use the [BulkTracking api](http://developer.piwik.org/api-reference/tracking-api#bulk-tracking).